app/modules/training/http_training.py

```diff
@@ -0,0 +1,55 @@
+import webapp2
+import logging
+import os
+import requests
+from google.appengine.ext.webapp import template
+from requests_toolbelt.adapters import appengine
+
+# https://toolbelt.readthedocs.io/en/latest/adapters.html#appengineadapter
+appengine.monkeypatch(validate_certificate=False)
+
+class HttpTrainingHandler(webapp2.RequestHandler):
+    def get(self, lat = None, lon = None):
+        logging.info("HttpTrainingHandler get(%s,%s)" % (lat, lon))
+
+        # wikipedia base api url
+        API_HOST = "http://en.wikipedia.org/w/api.php?"
```

Comment:
@Najipian Please change this URL to start with `https://` to avoid an additional HTTP request while Wikipedia redirects us to the https URL.